Test for search playstation #20

Natali-QA opened the pull request:
- added model for playstation product;
  - added test for searching playstation;
  - added json file for playstation product;

Natali-QA committed:
Test for search playstation #20

mkgerasimenko commented on src/main/java/io/github/sskorol/model/Playstation.java:
```diff
@@ -0,0 +1,16 @@
+    public Playstation(final String name, final String generalCategory, final String purchaseStatus) {
+        super(name, generalCategory, purchaseStatus);
+    }
+}
```
please add the new line.

Natali-QA committed:
Test for search playstation #20
  - added model for playstation product;
  - added test for searching playstation;
  - added json file for playstation product;
